config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  get 'schedules/index'
+  get 'schedules/update'
+  get 'schedules/edit'
+  get 'schedules/new'
+  get 'schedules/show'
```

Comment:
If we're nesting the schedule routes under kurasus, we don't need them here.

Reply:
Fixed!